packages/nimbus/src/components/tabs/tabs.recipe.tsx

```diff
@@ -0,0 +1,216 @@
+import { defineSlotRecipe } from "@chakra-ui/react/styled-system";
+
+/**
+ * Recipe configuration for the Tabs component.
+ * Defines the styling variants and base styles using Chakra UI's recipe system.
+ */
+export const tabsSlotRecipe = defineSlotRecipe({
+  slots: ["root", "list", "tab", "panels", "panel", "trigger"],
+
+  className: "nimbus-tabs",
+
+  base: {
+    root: {
+      display: "flex",
+      width: "100%",
+    },
+    list: {
+      display: "flex",
+      flexShrink: 0,
+    },
+    tab: {
+      appearance: "none",
+      background: "transparent",
+      border: "none",
+      color: "neutral.12",
+      cursor: "pointer",
+      display: "flex",
+      alignItems: "center",
+      justifyContent: "center",
+      gap: "200",
+      fontWeight: "500",
+      padding: "300 400",
+      transition: "all 150ms ease",
+      paddingTop: "var(--taps-adding-top)",
+      paddingRight: "var(--tabs-padding-right)",
+      paddingBottom: "var(--tabs-padding-bottom)",
+      paddingLeft: "var(--tabs-padding-left)",
+      fontSize: "var(--tabs-font-size)",
+      _hover: {
+        color: "primary.11",
+      },
+      _selected: {
+        color: "primary.9",
+        borderColor: "primary.9",
+      },
+      _disabled: {
+        layerStyle: "disabled",
+      },
+      _focusVisible: {
+        outlineWidth: "var(--focus-ring-width)",
+        outlineColor: "var(--focus-ring-color)",
+        outlineStyle: "var(--focus-ring-style)",
+        outlineOffset: "2px",
```

Comment:
I believe that these styles can be replaced with `layerStyle: "focusRing"`, [which was added recently](https://github.com/commercetools/nimbus/blob/1e9f91c90cd69443ae728bdb73449de4be52e930/packages/nimbus/src/theme/layer-styles.ts#L12)